x-pack/plugins/stack_connectors/public/connector_types/cases_webhook/severity_filter.tsx

```diff
@@ -23,31 +24,51 @@ export enum CaseSeverity {
   CRITICAL = 'critical',
 }
 
-export const severities = {
-  [CaseSeverity.LOW]: {
-    color: euiLightVars.euiColorVis0,
-    label: i18n.SEVERITY_LOW_LABEL,
-  },
-  [CaseSeverity.MEDIUM]: {
-    color: euiLightVars.euiColorVis5,
-    label: i18n.SEVERITY_MEDIUM_LABEL,
-  },
-  [CaseSeverity.HIGH]: {
-    color: euiLightVars.euiColorVis7,
-    label: i18n.SEVERITY_HIGH_LABEL,
-  },
-  [CaseSeverity.CRITICAL]: {
-    color: euiLightVars.euiColorVis9,
-    label: i18n.SEVERITY_CRITICAL_LABEL,
-  },
-};
+// export const severities = {
+//   [CaseSeverity.LOW]: {
+//     color: euiLightVars.euiColorVis0,
+//     label: i18n.SEVERITY_LOW_LABEL,
+//   },
+//   [CaseSeverity.MEDIUM]: {
+//     color: euiLightVars.euiColorVis5,
+//     label: i18n.SEVERITY_MEDIUM_LABEL,
+//   },
+//   [CaseSeverity.HIGH]: {
+//     color: euiLightVars.euiColorVis7,
+//     label: i18n.SEVERITY_HIGH_LABEL,
+//   },
+//   [CaseSeverity.CRITICAL]: {
+//     color: euiLightVars.euiColorVis9,
+//     label: i18n.SEVERITY_CRITICAL_LABEL,
+//   },
+// };
 
 interface Props {
   selectedSeverity: CaseSeverity;
   onSeverityChange: (status: CaseSeverity) => void;
 }
 
 export const SeverityFilter: React.FC<Props> = ({ selectedSeverity, onSeverityChange }) => {
+  const { euiTheme } = useEuiTheme();
```

Comment:
I guess this will be removed in favour of the SeverityHealth component?

Reply:
removed it and used the new SeverityHealth component